src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -331,29 +337,29 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
       <br />
       {numApps > 0 && (
         <>
-          <Icon name="applications" type="pf" style={{ padding: '0 1em' }} />
+          <KialiIcon.Applications className={topologyStyle} />
```

Comment:
Hmm... for some reason, these are not showing correctly:

![image](https://user-images.githubusercontent.com/23639005/67591997-4e891200-f724-11e9-9d7b-d03dcf4b610e.png)

Reply:
![Kiali_Console_-__Private_Browsing_](https://user-images.githubusercontent.com/1312165/67601186-9b6ee780-f728-11e9-9c26-fe33221bc3bf.png)
Ok fixed. The padding was squishing the icons, so I changed to 1em margin.